compose-builder/add-security.yml

```diff
@@ -154,87 +153,45 @@ services:
     restart: always
 
 # containers for reverse proxy
-  kong-db:
-    image: postgres:${POSTGRES_VERSION}
-    user: "root:root" # Note that postgres is run under the 'postgres' user, but entry point scripts need to first run as root
-    container_name: edgex-kong-db
-    hostname: edgex-kong-db
-    read_only: true
-    restart: always
-    networks:
-      - edgex-network
-    ports:
-        - "127.0.0.1:5432:5432"
-    tmpfs:
-      - /var/run
-      - /tmp
-      - /run
-    volumes:
-      - edgex-init:/edgex-init:ro,z
-      - postgres-data:/var/lib/postgresql/data:z
-      - postgres-config:/tmp/postgres-config:z
-    entrypoint: ["/edgex-init/postgres_wait_install.sh"]
-    env_file:
-      # needed for vault host/port
-      - common-security.env
-      - common-sec-stage-gate.env
-    environment:
-      POSTGRES_DB: kong
-      POSTGRES_USER: kong
-      POSTGRES_PASSWORD_FILE: "${POSTGRES_PASSWORD_FILE:-/tmp/postgres-config/.pgpassword}"
-    depends_on:
-      - security-bootstrapper
-    security_opt: 
-      - no-new-privileges:true
 
-  kong:
-    image: kong:${KONG_VERSION}
-    user: "kong:nogroup" # Note Kong already running as kong user but doesn't define a group
-    container_name: edgex-kong
-    hostname: edgex-kong
+  nginx:
+    image: nginx:alpine-slim
```

Comment:
Please put the image version in the .env file as NGINX_VERSION and actually pin it to a minor version.

Reply:
Done